core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/machine/ServerProperties.java

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A., Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Mario Loriedo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.core.model.machine;
+
+import org.eclipse.che.commons.annotation.Nullable;
+
+/**
+ * Not mandatory properties of a {@link Server}
+ *
+ * @author Mario Loriedo
+ */
+public interface ServerProperties {
+
+    /**
+     * Path to access the server.
+     */
+    @Nullable
+    String getPath();
+
+    /**
+     * Internal address of the server in form <b>host:port</b>
```

Comment:
Line break by entering 'enter' key doesn't work in javadocs. Please add dot and `<p/>` or `<br/>`